Marlin/src/libs/MAX31865.cpp

```diff
@@ -458,19 +459,23 @@ void MAX31865::writeRegister8(uint8_t addr, uint8_t data) {
   if (_sclk == TERN(LARGE_PINMAP, -1UL, -1))
     SPI.beginTransaction(spiConfig);
   else
-    WRITE(_sclk, LOW);
+    write(_sclk, LOW);
 
-  WRITE(_cs, LOW);
+  write(_cs, LOW);
 
   spixfer(addr | 0x80); // make sure top bit is set
   spixfer(data);
 
   if (_sclk == TERN(LARGE_PINMAP, -1UL, -1))
     SPI.endTransaction();
 
-  WRITE(_cs, HIGH);
+  write(_cs, HIGH);
 }
 
+void MAX31865::write(pin_t pin, bool value){
+  for (uint8_t i = 0; i < _spi_speed; i++)
+    WRITE(pin, value);
+}
```

Comment:
Does the SoftwareSPI library have a means of doing this, instead of calling `WRITE()` `_spi_speed` times? Perhaps also, instead of calling write in a loop, a `safe_delay()` call could be used here.

Reply:
Note that FastIO macros like `WRITE` and `OUT_WRITE` require the first argument to be known at compile-time. They cannot take a variable, only a pin name or equivalent preprocessor define. If the pin is a variable, then the code must use `pinMode`, `digital_write`, etc.

Reply:
@thinkyhead does this mean that calling `WRITE` this way does not actually do anything?